### ISSUE TITLE
Keyword for retrieving Browser console logs

### ISSUE THREAD

**Issue by kvo-harmoney** [MarketSquare/robotframework-browser#532]
Selenium offers the possibility to fetch its browser console logs in a following way :

`    def get_browser_logs(self):
        seleniumlib = BuiltIn().get_library_instance('SeleniumLibrary')
        browser_logs = seleniumlib.driver.get_log('browser')`

For the new browser library approach, there is no such thing available yet, although that playwright itself offers some classes which might do the trick ( consoleMessage ).

Request : provide keyword so that in an easy way the console logs of a session can be captured, and used for post-UI-run processing.

**Comment by mkorpela** [comment]
Experimented with this for a while. This PoC somewhat works but it is not yet enduser ready solution.
I'll anyway show it here for now for further work.

extension.js
```javascript
async function pipeLogToRF(page) {
   let messages = [];
   page.on('console', message => {
       console.log(message);
       messages.push(message.text());
   });
   await new Promise((resolve) => {
    setTimeout(() => resolve(), 5000)
    });
   return messages;
}

exports.__esModule = true;
exports.pipeLogToRF = pipeLogToRF;
```

```RobotFramework
*** Settings ***
Library   Browser  jsextension=${CURDIR}/extension.js

*** Test Cases ***
Example Test
   Open Browser  https://www.reaktor.com
   ${p}=  Promise To  pipeLogToRF
   Click  a[data-title="Work"]
   Sleep  1
   Take Screenshot
   Go to  https://www.hs.fi
   wait until network is idle
   ${data}=  wait for  ${p}
```

**Comment by aaltat** [comment]
Closing because  #745 is done. Please tell us if something is missing from the new feature implemented in  #745
